cli/lsp/language_server.rs

```diff
@@ -602,6 +603,17 @@ impl Inner {
   pub async fn update_import_map(&mut self) -> Result<(), AnyError> {
     let mark = self.performance.mark("update_import_map", None::<()>);
 
+    if let Some(config_file) = &self.maybe_config_file {
+      if let Some(import_map_json) = config_file.to_import_map() {
+        let import_map_url =
+          self.maybe_config_file.as_ref().unwrap().specifier.clone();
+        let import_map =
```

Comment:
We should discuss this - not sure if import map from the config file should have precedence over `deno.importMap` setting in VSCode

Reply:
Discussed with the team, `deno.importMap` should have precedence over import map in the config file.